.circleci/config.yml

```diff
@@ -36,6 +36,46 @@ commands:
           paths:
             - node_modules
 
+  build-win:
```

Comment:
This is unrelated to these changes. Your extension is simpler than the salesforcedx-vscode and does not have the need of using lerna to build it. You should look into using yarn instead of npm since it's faster, provides better lock file functionality, among other things.

Reply:
I'll look into using yarn instead of npm for a future release. Thanks!